cns/NetworkContainerContract.go

```diff
@@ -338,26 +340,47 @@ type CreateNetworkContainerResponse struct {
 	Response Response
 }
 
-// GetNetworkContainerStatusRequest specifies the details about the request to retrieve status of a specifc network container.
+// GetNetworkContainerStatusRequest specifies the details about the request to retrieve status of a specific network container.
 type GetNetworkContainerStatusRequest struct {
 	NetworkContainerid string
 }
 
-// GetNetworkContainerStatusResponse specifies response of retriving a network container status.
+// GetNetworkContainerStatusResponse specifies response of retrieving a network container status.
 type GetNetworkContainerStatusResponse struct {
 	NetworkContainerid string
 	Version            string
 	AzureHostVersion   string
 	Response           Response
 }
 
-// GetNetworkContainerRequest specifies the details about the request to retrieve a specifc network container.
+// GetAllNetworkContainersRequest specifies the details about the request to retrieve all network containers on CNS.
+type GetAllNetworkContainersRequest struct {
+	NodeID string
+}
```

Comment:
If I'm understanding the flow correctly, DNC will send a `GET` request to CNS to obtain all the NCs' details in the response. Since we are looping over the nodes on DNC (one call per node), initially I was thinking to retrieve all the NCs on a particular node by node ID.
What I am not understand here is that what does the `check for the http method in the handler` refer to? And what does the request body from DNC look like?
For now, I changed it to `ncIDs[]` in the request. The details of each NC in the CNS will be returned by the NC ID.